<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string> 主界面</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>171</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>171</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-20</y>
         <width>800</width>
         <height>551</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string notr="true"> 数据接收</string>
        </attribute>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>72</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string> 端口号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>72</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string> 波特率：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>72</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>  数据位：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>72</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>  校验位：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>230</y>
           <width>72</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string> 停止位：</string>
         </property>
        </widget>
        <widget class="QComboBox" name="PortBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="BaudBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="DataBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>130</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="ParityBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="StopBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>230</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
        <widget class="QTextEdit" name="writeEdit_3">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>391</width>
           <height>281</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="sendEdit_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>320</y>
           <width>191</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSend_3">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnClear_3">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string> 清空</string>
         </property>
        </widget>
        <widget class="QLabel" name="portlabel_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>340</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnOpen_3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>340</y>
           <width>91</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string notr="true">历史数据</string>
        </attribute>
        <widget class="QTableView" name="tableView_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>31</y>
           <width>551</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>  TCP/IP</string>
        </attribute>
        <widget class="QLabel" name="wenshiLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>211</width>
           <height>271</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="yanwuLabel">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>60</y>
           <width>191</width>
           <height>191</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>300</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string> 异常记录：</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>350</y>
           <width>251</width>
           <height>131</height>
          </rect>
         </property>
         <property name="title">
          <string> 当前信息</string>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string> 温度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string> 湿度</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="wendu">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_24">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>20</y>
            <width>41</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string> ℃</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="shidu">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>80</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>80</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string> %</string>
          </property>
         </widget>
        </widget>
        <widget class="QTextEdit" name="errorEdit">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>340</y>
           <width>281</width>
           <height>171</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="clear">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string> 清空</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string notr="true"> 数据接收</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string> 端口号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string> 波特率：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>  数据位：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>  校验位：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string> 停止位：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PortBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="DataBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="ParityBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>180</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="StopBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QTextEdit" name="writeEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>561</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="sendEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>320</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSend">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string> 清空</string>
       </property>
      </widget>
      <widget class="QLabel" name="portlabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOpen">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>340</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string notr="true">历史数据</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>31</y>
         <width>741</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string> 温湿度实时曲线图</string>
      </attribute>
      <widget class="QWidget" name="w_widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>711</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
